src/test/java/seedu/address/testutil/SampleDataUtilTest.java

```diff
@@ -0,0 +1,28 @@
+package seedu.address.testutil;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertNotNull;
+
+import org.junit.jupiter.api.Test;
+
+import seedu.address.model.AddressBook;
+import seedu.address.model.person.Person;
+import seedu.address.model.util.SampleDataUtil;
+
+public class SampleDataUtilTest {
+    @Test
+    public void getSamplePersons_validData_personsArray() {
+        // Test that sample persons are correctly created
+        Person[] samplePersons = SampleDataUtil.getSamplePersons();
+        assertNotNull(samplePersons);
+        assertEquals(6, samplePersons.length); // Check the array size
+    }
```

Comment:
Im not really sure if this is needed since `getSampleAddressBook` is already calling it during test. It will be hard to make the sample size flexible as well.

Reply:
Alright got it! I will review it and change it accordingly! Thank you!